tox.ini

```diff
@@ -3,7 +3,7 @@
 
 [tox]
 envlist =
-    py{38,39}-internal-{cpu,cuda,cuda90,cuda91,cuda92,cuda100,cuda101}
+    py{38,39}-{internal,all}-{cpu,cuda,cuda90,cuda91,cuda92,cuda100,cuda101}
```

Comment:
Everywhere else in this PR the new factor is called `with-externals` instead of `all`, so it should be updated accordingly here. However, `with-externals` contains a dash and most likely is going to mess up with `tox` matrix/factors system, so it should be replaced with something else (ideally a single word, but probably underscores would be also fine).

Reply:
I don't remember why I switched from all to with-externals (even it was only 2 hours ago). I'll go back to all.